hardware/openwebnet/bt_openwebnet.cpp

```diff
@@ -896,27 +896,19 @@ bool bt_openwebnet::IsEqual(const bt_openwebnet& msg_to_compare)
 
   if(!m_extended)
   {
-    if ((msg_to_compare.Extract_who().compare(m_who) == 0) &&
-        (msg_to_compare.Extract_what().compare(m_what) == 0) &&
-        (msg_to_compare.Extract_where().compare(m_where) == 0) &&
-        (msg_to_compare.Extract_when().compare(m_when) == 0) &&
-        (msg_to_compare.Extract_dimension().compare(m_dimension) == 0))
-      return true;
-    else
-      return false;
+	  if ((msg_to_compare.Extract_who() == m_who) && (msg_to_compare.Extract_what() == m_what) && (msg_to_compare.Extract_where() == m_where) && (msg_to_compare.Extract_when() == m_when) &&
+	      (msg_to_compare.Extract_dimension() == m_dimension))
+		  return true;
```

Comment:
would be better if it was
if (
(a == 1)
|| (b==1)
|| (c==1)
)

Reply:
The original check looks much cleaner, except it used string.compare, and i would place && at the start